docs/java-compatibility.adoc

```diff
@@ -0,0 +1,6 @@
+= Java Compatibility with TomEE 8.0.x
+:index-group: General Information
+:jbake-type: page
+:jbake-status: published
+
+TomEE 8.0.x is Java EE 7, Java 8 & Java 10. Any OS that supports Java 10 works
```

Comment:
change Java EE 7 to Java EE 8

Reply:
Java 10 ended support in September 2018, there is no more release for it.

Reply:
Updated with correct version 
Thanks